Weather App/weather_utils.py

```diff
@@ -0,0 +1,72 @@
+from datetime import datetime
+
+class WeatherUtils():
+    
+    @staticmethod
+    def parse_location(data):
+        coordinate = (data["latt"], data["longt"])
+        location = f'{data["standard"]["city"]}, {data["standard"]["countryname"]}'
+        return {"coordinate":coordinate, "location":location}
+    
+    @staticmethod
+    def json_convert(resBoth):
```

Comment:
resBoth - not pythonic namimg. will rename by myself after to res_both.
https://peps.python.org/pep-0008/#function-and-variable-names